src/test/cypress/integration/exam/ExamDateVerification.spec.ts

```diff
@@ -91,7 +91,7 @@ describe('Exam management', () => {
                         examStartEnd.startExam();
                         cy.contains('Exam Overview').should('exist');
                         cy.contains(textExercise.title).should('be.visible').click();
-                        cy.get('#text-editor-tab').type(
+                        artemis.pageobjects.textEditor.typeSubmission(
                             'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.',
```

Comment:
<a href="https://github.com/ls1intum/Artemis/pull/3918#discussion_r706851196"> Here </a> you asked me to create a textfile or a json file which would contain our text exercise submissions. I think it would also be prudent to do this here instead of having this long string in the code.

Reply:
Sure makes sense, but that shouldn't be part of this PR since that code section wasn't added by my PR. I will add it in the [text exercise assessment PR](https://github.com/ls1intum/Artemis/pull/3963)

Reply:
I added this in this PR nevertheless because another spec file needed the fixture as well